### PR TITLE
added ngx bulma logo

### PR DESCRIPTION
**Pull request by santoshyadavdev**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -140,6 +140,13 @@
       "description": "Angular Kenya Community Logo",
       "license": "MIT",
       "name": "Angular Kenya"
+    },
+    "ngx-bulma": {
+      "name": "Team ngx-builders",
```
Comment (fmalcher) [inline metadata.json]:
```suggestion
      "name": "ngx-bulma"
```

File: metadata.json [hunk 2]
```diff
@@ -140,6 +140,13 @@
...
+      "creator": "ngx-builders",
+      "description": "ngx-builders team",
+      "license": "MIT",
+      "website": "https://ngx2-bulma.netlify.app/"
```
Comment (fmalcher) [inline metadata.json]:
```suggestion
      "website": "https://ngx2-bulma.netlify.app"
```

**Comment by fmalcher** [comment]
Thank you, @santoshyadavdev !
Unfortunately, the logo is an SVG with embedded PNG. Do you have an opportunity to convert it to vectors?

**Comment by santoshyadavdev** [comment]
I used some online tool, which was free. Can you guide me which tool I can use to covert.

**Comment by fmalcher** [comment]
Do you have the source file from that online tool or did it just give out an SVG for you and nothing else?

**Comment by santoshyadavdev** [comment]
It gave me svg

**Comment by santoshyadavdev** [comment]
I will try tomorrow, to convert it again.

**Comment by fmalcher** [comment]
I recreated it in vectors, was just 2 minutes 😅 https://www.dropbox.com/s/1zbix4zj3v4b69r/bulma.svg?dl=1
@santoshyadavdev

**Comment by santoshyadavdev** [comment]
Thank you @fmalcher pushed the latest image. Thanks again for your help.

File: metadata.json [hunk 3]
```diff
@@ -140,6 +140,13 @@
       "description": "Angular Kenya Community Logo",
       "license": "MIT",
       "name": "Angular Kenya"
+    },
+    "ngx-bulma": {
+      "name": "ngx-bulma",
+      "creator": "ngx-builders",
+      "description": "ngx-builders team",
```
Comment (fmalcher) [inline metadata.json]:
```suggestion
      "description": "Logo of ngx-bulma – the Angular Library with Bulma CSS",
```

**Review by fmalcher — APPROVED** [review]
LGTM

**Comment by maartentibau** [comment]
Thank you! 🙏

**Pull request merged**